sdks/python/conftest.py

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
```

Comment:
yes. conftest.py is a local plugin for Pytest and contains self-defined hook implementation. pytest loads this plugin at tool startup and some hooks (like pytest_runtest_setup()) run before each test method call. Put this file in ../apache_beam directory is fine as long as the directory contains all tests that we want this plugin covers. 

I'll add comments as well as explain new added command line options.